python/tests/test_variable.py

```diff
@@ -163,14 +167,216 @@ def test_create_1D_vector_3_float64():
     assert var.unit == sc.units.m
 
 
-def test_create_datetime64():
-    var = sc.Variable(dims=['x'],
-                      unit=sc.units.ns,
-                      shape=[2],
-                      dtype=sc.dtype.datetime64)
-    assert var.dims == ['x']
-    assert var.dtype == sc.dtype.datetime64
-    assert var.unit == sc.units.ns
+def test_construct_0d_datetime():
```

Comment:
Consider putting all datetime test in a separate file, since there are many and they form a nice unit.